.rubocop.yml

```diff
@@ -1,5 +1,8 @@
 AllCops:
-  TargetRubyVersion: 2.3
+  TargetRubyVersion: 2.4
```

Comment:
Ruby 2.4 and 2.5 are both EOL. 2.6 will be EOL soon-ish. I think we can bump this to a more modern ruby.

Reply:
Yep, good call, bumped to 2.7.

Reply:
Bumped back to 2.6, just since that's technically still a supported version and Rubocop wants us to keep the target version and the supported version in sync (which makes sense).